pkg/decode/decode_protobuf.go

```diff
@@ -69,12 +69,19 @@ func RecordToMap(fr *model.Record) config.GenericMap {
 
 	var directions []int
 	var interfaces []string
+	var udns []string
 	for _, intf := range fr.Interfaces {
 		directions = append(directions, intf.Direction)
 		interfaces = append(interfaces, intf.Interface)
+		if intf.Udn != "" {
```

Comment:
shouldn't we append udns here, regardless of it's presence? It depends how this data is processed in the console plugin. If the udn array is correlated with the interface array, the indexes in udn array must match the one from the interfaces. Not totally sure about that.

Reply:
in the console side they are two separate list? u mean when user correlate between the two in the console ui they won't be in sync so maybe its better to show it regardless I will remove this check

Reply:
honestly I don't remember how it's one in the console plugin ... For interfaces and direction I know that we have a column that shows both "Interfaces and Directions" so the 2 arrays are correlated; but did we do something similar for udn, I'm not sure